docs/get-started/alerting.md

```diff
@@ -8,62 +8,163 @@ Alerting notifies of important or unusual activity in your database environments
-- **Mimir or Loki recording rules**: precompute the result of expensive queries and execute alerts faster. 
-With Mimir and Loki alert rules, you can run alert expressions closer to your data and at massive scale, managed by Grafana. 
+- **Mimir or Loki recording rules**: precompute the result of expensive queries and execute alerts faster.
+With Mimir and Loki alert rules, you can run alert expressions closer to your data and at massive scale, managed by Grafana.
+
+-->
 
 ## Alerting components
-Alerts are split into four key components: alert rules, contact points, notification policies, and silences. 
+Alerts are split into four key components: alert rules, contact points, notification policies, and silences.
 
 ### Alert rules
-Describe the circumstances under which you want to be alerted. The evaluation criteria that you define determine whether an alert will fire. 
+Describe the circumstances under which you want to be alerted. The evaluation criteria that you define determine whether an alert will fire.
 
 An alert rule consists of one or more queries and expressions, a condition, the frequency of evaluation, and the duration over which the condition is met.
 
 For example, you might configure an alert to identify and notify you when MongoDB is down.
 
-Provide a simplified framework for configuring complex alert rules. 
+Provide a simplified framework for configuring complex alert rules.
 
 PMM includes a set of default templates with common events and expressions for alerting. You can also create your own templates if you need custom expressions on which to base your alert rules.
 
-You can check the  alert templates available for your account under **Alerting > Alert rule templates** tab. PMM lists here the following types of templates:
+You can check the alert templates available for your account under **Alerting > Alert rule templates** tab. PMM lists here the following types of templates:
 
 1. Built-in templates, available out-of-the-box with PMM.
-2. Custom templates created or uploaded on the **Alerting page > Alert Templates** Tab. 
-3. Custom template files available in your  ``yaml srv/alerting/templates`` directory. PMM loads them during startup.
-
-### Silences
-Silences specify periods of time to suppress alerts and their associated notifications. During a silence, PMM continues to track metrics and trigger alerts but does not send notifications to the specified contact points. Once the silence expires alerts and notifications will resume.
-For example, you can create a silence to suppress trivial notifications during weekends.
+2. Custom templates created or uploaded on the **Alerting page > Alert Templates** tab.
+3. Custom template files available in your ``srv/alerting/templates`` directory. PMM loads them during startup.
```

Comment:
I dont see the reason to state #3 as a separate item...why make this distinction here? You start by saying/listing what is visible on the templates page...#3 is not, that I can see. If you feel this knowledge is necessary,  

OR...you could append point #2 to say: 

Custom templates created or uploaded on the **Alerting page > Alert Templates** tab. Custom template files are stored in the ``srv/alerting/templates`` directory. PMM loads them during startup."